dashboard/app/controllers/javabuilder_sessions_controller.rb

```diff
@@ -8,9 +8,12 @@ class JavabuilderSessionsController < ApplicationController
 
   PRIVATE_KEY = CDO.javabuilder_private_key
   PASSWORD = CDO.javabuilder_key_password
+  CSA_PILOT = 'csa-pilot'
+  CSA_PILOT_FACILITATORS = 'csa-pilot-facilitators'
```

Comment:
Is there a variable somewhere that is the global source of truth for these? Mostly thinking about how we find all instances of this once the pilot is over.

Reply:
Unfortunately I think the source of truth is in the course file: https://github.com/code-dot-org/code-dot-org/blob/e0794200eddbc8697ace8c940cc0af11ed6b4968/dashboard/config/courses/csa-pilot.course#L19